src/resolvers/Mutation/issue.js

```diff
@@ -0,0 +1,27 @@
+const issue = {
+  createIssue: async (parent, { name, description }, ctx, info) => {
+    return ctx.db.mutation.createIssue({
```

Comment:
no need await here?

Reply:
@giathinh910 Yea, because we have only one async action here. So `prisma-binding` will handle this automatically for us.

Reply:
thanks (y)